corehq/apps/locations/forms.py

```diff
@@ -290,6 +291,10 @@ def _sync_location_user(self):
             if user:
                 user.is_active = False
                 user.save()
+                log_user_change(by_domain=self.domain, for_domain=user.domain,
+                                couch_user=user, changed_by_user=self.user,
+                                changed_via=USER_CHANGE_VIA_LOCATION,
+                                fields_changed={'is_active': user.is_active})
```

Comment:
Good to audit, but FYI I believe this is a dead feature.  This only applies to "location users," which was a thing where we auto-create a user that's 1-to-1 with a location.  Looks like that code was partially removed in https://github.com/dimagi/commcare-hq/pull/24055, and this seems like a remnant that doesn't make much sense without the rest.

Reply:
Just ran a query and found:

* `www` has 1 such location, on the `dimagi` domain
* `india` has none
* `staging` has 11 such locations, on three domains

Sounds pretty killable to me